fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -778,8 +758,8 @@ public boolean isPreviousTransactionsFinished(long endTransactionId, long dbId)
                     continue;
                 }
                 if (entry.getKey() <= endTransactionId) {
-                    LOG.debug("find a running txn with txn_id={}, less than schema change txn_id {}", 
-                            entry.getKey(), endTransactionId);
+                    LOG.info("find a running txn with txn_id={} on db: {}, less than watermark txn_id {}",
```

Comment:
should check at tablet level not at db level

Reply:
This PR is already too big, so I will improve this at another PR. I will create an issue for it #1724